src/hypervisor.c

```diff
@@ -107,6 +108,43 @@ cc_oci_expand_net_device_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
+/*
+ *
+ *
+ */
+#define DPDK_CMDLINE_OBJ "memory-backend-file,id=dpdkmem,size=2048M,mem-path=/dev/hugepages,share=on,prealloc=on"
```

Comment:
While that would work, the need to read it from a file and parse out that particular subset of a string on a particular line makes me less than enthusiastic. :)

OVS DPDK is an optional thing to do, so I understand not wanting to necessarily change master's default behavior.

The 2GB is pretty arbitrary choice, right?  Is this something we are likely to want to change over time (i.e., should we make this more configurable than just in the file?)?